lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -1132,7 +1391,12 @@ class MongoClientInterface {
                 MongoUtils.unserialize(doc.value);
                 return next(null, doc.value);
             },
-        ], cb);
+        ], (err: ArsenalError | null | undefined, result?: ObjectMDData) => {
+            if (err) {
+                return cb(err);
+            }
+            return cb(null, result!);
+        });
```

Comment:
can't anymore pass `cb` directly here, it fails type-check?

Reply:
Yes, the async type is not matching (`AsyncResultCallback<unknown, ArsenalError>`).